tests/MongoDB.Driver.Tests/Specifications/command-monitoring/TestRunner.cs

```diff
@@ -104,6 +106,8 @@ public bool OneTimeSetup()
         [ClassData(typeof(TestCaseFactory))]
         public void RunTestDefinition(IEnumerable<BsonDocument> data, string databaseName, string collectionName, BsonDocument definition, bool async)
         {
+            RequireServer.Check().ClusterTypes(ClusterType.ReplicaSet, ClusterType.Standalone);
```

Comment:
Nit: could make this a bit less exclusionary by allowing it to run on pre 3.7 sharded clusters.  Your call though.